app/models/notifier.rb

```diff
@@ -168,12 +170,24 @@ def changeset_comment_notification(comment, recipient)
                   I18n.t("notifier.changeset_comment_notification.commented.subject_other", :commenter => @commenter)
                 end
 
+      attachments.inline["logo.png"] = File.read(Rails.root.join("app", "assets", "images", "osm_logo_30.png"))
+      attachments.inline["avatar.png"] = File.read(user_avatar_file_path(comment.author))
+
       mail :to => recipient.email, :subject => subject
     end
   end
 
   private
 
+  def user_avatar_file_path(user)
+    image = user.image
+    if image.file?
+      return image.path.sub("/original/", "/small/")
+    else
+      return Rails.root.join("app", "assets", "images", "users", "images", "small.png")
+    end
+  end
+
```

Comment:
`image.path(:small)` works, thanks.

However `image_path("users/images/small.png")` returns `"/assets/users/images/small-19cb2e87cebaee3dbe77d1b540f68136dae6f8e63ec055907d19f639773101e2.png"`, which is the path part of a public URL. But I want something that I can `File.read`.

Would you find the above code less objectionable if I just concatenated it all into one string using slashes, e.g. `"#{Rails.root}/app/assets/images/users/images/small.png"` ? That seems more in line with what other parts of the code already do (for config files, though, not for asset files).